### DISCUSSION TITLE
Github Copilot with Pycharm rocks!

### DISCUSSION THREAD

**Discussion by Denis2054** [community/community#6985] [Copilot Conversations]
Incredible!

I got it working in a few minutes!

Since I wrote a book on Transformers, Transformers for NLP, 2021, Denis Rothman (just type my name on Amazon), I see how to engineer prompts to get what I want!

I recommend understanding Transformers and prompt engineering to get the best out of this fantastic tool!

**Comment by DeqianBai** [comment]
could you post your example here

**Reply by Denis2054** [reply]
GitHub copilot code with PyCharm is not a stand alone program.
In Chapter 16 of Transformers for NLP, 2nd Edition, I take the reader through the process step by step to generate code automatically locally(on a laptop, for example).
It sure is fun!
